packages/viewer/src/components/viewer/viewer.tsx

```diff
@@ -389,6 +390,23 @@ export class Viewer {
     return this.stream;
   }
 
+  /**
+   * @private Used for testing.
+   */
+  public async handleWebSocketClose(): Promise<void> {
```

Comment:
just making public to test?

Reply:
Yep, that's correct. Wanted to be able to hook into this function on disposal of the StreamApi, similar to how it'd get triggered when the WebSocket is closed through that disposal for the test